src/main/java/org/jabref/gui/DialogService.java

```diff
@@ -26,7 +26,7 @@
 public interface DialogService {
 
     /**
-     * This will create and display new {@link ChoiceDialog} of type T with a default choice and a collection of possible choices
+     * This will create and display new {@link ChoiceDialog} of type T with a default choice (can be null) and a collection of possible choices
```

Comment:
mhhh, we wanted to prevent as much `null` values as possible. It's probably better to add a new default method in the interface without the `defaultChoice` (and passing `null` to the overload). In this way, the null argument only occurs in one point and not everywhere in the code.